src/borg/helpers.py

```diff
@@ -1191,6 +1192,20 @@ def yes(msg=None, false_msg=None, true_msg=None, default_msg=None,
         env_var_override = None
 
 
+def ellipsis_truncate(msg, space):
+    from .platform import swidth
+    """
+    shorten a long string by adding ellipsis between it and return it, example:
+    this_is_a_very_long_string -------> this_is..._string
+    """
```

Comment:
please put the docstring right under the def, not into the middle of the function.

was the switdth import circular or why don't you do it on module level?